app/pulp/app/models/progress.py

```diff
@@ -181,15 +181,6 @@ class ProgressBar(ProgressReport):
         >>> with progress_bar:
         >>>     for file in progress_bar.iter(files_iterator):
         >>>         handle(file)
-
-    You can also use this short form:
-
-        >>> with ProgressBar(message='Publishing files', total=len(files_iterator)):
```

Comment:
Maybe this should be?

`with ProgressBar.objects.create(message='Publishing files', total=len(files_iterator)):`